src/main/java/org/aksw/simba/squirrel/frontier/impl/FrontierImpl.java

```diff
@@ -60,38 +50,107 @@ public UriQueue getQueue() {
      */
     protected GraphLogger graphLogger;
 
+
+    /**
+     * Indicates whether recrawling is active.
+     */
+    private boolean doesRecrawling;
+
+    /**
+     * The timer that schedules the recrawling.
+     */
+    private Timer timerRecrawling;
+
+    /**
+     * Time (in milliseconds) after which uris will be recrawled.
+     */
+    private Long recrawlTime;
+
+    /**
+     * Time interval(in milliseconds) at which the check for outdated uris is performed.
+     */
+    private Long timerPeriod;
+
+    /**
+     * General time (in milliseconds) after which uris will be recrawled, if no other time is specified.
+     */
+    private static long generalRecrawlTime = 1000 * 60 * 60 * 24 * 7;
```

Comment:
This value shouldn't be static since it is defined for the single frontiers (although you typically wouldn't have more than one frontier).

Reply:
@MichaelRoeder yes, but the problem is that we need to access this field from WorkerImpl which has no access to the concrete instance of FrontierImpl. Therefore, it is easier to make the field static.